modules/mod_twitter/support/twitter_import_tweet.erl

```diff
@@ -127,6 +128,7 @@ extract_import_rsc(TweetId, UniqueName, User, Tweet, Context) ->
 first_link([Url|_], _) -> Url;
 first_link(_, Url) -> Url.
 
+-spec first_media_props([#url_metadata{} | string() | binary()], #context{}) -> {error, nomedia} | {ok, pos_integer()}.
```

Comment:
The spec in line 131 uses a record, please define a type for the record and use that instead.